addons/mail/static/src/models/thread/thread.js

```diff
@@ -452,18 +459,32 @@ function factory(dependencies) {
 
         /**
          * Mark the specified conversation as read/seen.
+         * @param {mail.message} [message] the message to be considered as last seen, by default
```

Comment:
Method should take an ID as parameter, similar to everywhere else for the same reason (comparing ids, not actual message).